src/test/java/org/isf/visits/test/Tests.java

```diff
@@ -202,20 +202,6 @@ public void testMgrGetVisitsEmpty() throws Exception {
 		assertThat(visitManager.getVisits(0)).isEmpty();
 	}
 
-	@Test
-	public void testMgrGetVisitsAll() throws Exception {
-		int id = _setupTestVisit(false);
-		Visit visit = visitsIoOperationRepository.findOne(id);
-		assertThat(visitManager.getVisits(0)).hasSize(1);
-	}
-
-	@Test
-	public void testMgrGetVisitsbyID() throws Exception {
-		int id = _setupTestVisit(false);
-		Visit visit = visitsIoOperationRepository.findOne(id);
-		assertThat(visitManager.getVisits(id)).hasSize(1);
-	}
-
```

Comment:
is correct eliminating both?

Reply:
Yes; each one is duplicating another test elsewhere in the file.   Sorry I didn't make it more clear.